<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Fantasque Sans Mono</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TinyChat</string>
  </property>
  <widget class="QListWidget" name="msgListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="msgInputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="tabStopWidth">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
